css/styles.min.css

```diff
@@ -0,0 +1,2 @@
tent:center}.prompts-wrapper{display:-webkit-box;display:-ms-flexbox;display:flex;padding:10px 0}.prompts-wrapper-inner{padding:10px}.prompts-form{display:-webkit-box;display:-ms-flexbox;display:flex;-webkit-box-orient:vertical;-webkit-box-direction:normal;-ms-flex-direction:column;flex-direction:column;-webkit-box-align:center;-ms-flex-align:center;align-items:center}.prompts-form section{border:1px solid black;padding-top:10px}.prompts-radio-label{font-size:24px}.prompts-submit{padding-top:30px}.prompts-submit button{padding:5px 10px;font-family:inherit;font-size:20px}.prompts-players{margin-right:50px}.prompts-players,.prompts-team{display:-webkit-box;display:-ms-flexbox;display:flex;-webkit-box-orient:vertical;-webkit-box-direction:normal;-ms-flex-direction:column;flex-direction:column;-webkit-box-align:center;-ms-flex-align:center;align-items:center}.hidden{visibility:hidden}.disabled{pointer-events:none}body{font-family:"Courier New", Courier, monospace}h1{font-size:40px}h2{font-size:24px}.container{width:80%;margin:0 auto}.header{padding:20px 0;background-color:lightgrey}.game{display:-webkit-box;display:-ms-flexbox;display:flex;-webkit-box-orient:vertical;-webkit-box-direction:normal;-ms-flex-direction:column;flex-direction:column;-webkit-box-align:center;-ms-flex-align:center;align-items:center}.game-header{padding:20px 0}.game-grid{display:-ms-grid;display:grid;-ms-grid-columns:(1fr)[3];grid-template-columns:repeat(3, 1fr);-ms-grid-rows:(1fr)[3];grid-template-rows:repeat(3, 1fr);grid-column-gap:5px;grid-row-gap:5px;background-color:darkgray}.game-grid-wrapper{display:-webkit-box;display:-ms-flexbox;display:flex;-webkit-box-pack:center;-ms-flex-pack:center;justify-content:center;position:relative;width:-webkit-fit-content;width:-moz-fit-content;width:fit-content}.game-grid-item{border:1px solid black;height:95px;width:95px;display:-webkit-box;display:-ms-flexbox;display:flex;-webkit-box-pack:center;-ms-flex-pack:center;justify-content:center;-webkit-box-align:center;-ms-flex-align:center;align-items:center;background-color:white;cursor:pointer;font-size:40px}.game-grid-item:hover{background-color:lightgrey}.game-over{position:absolute;display:-webkit-box;display:-ms-flexbox;display:flex;-webkit-box-orient:vertical;-webkit-box-direction:normal;-ms-flex-direction:column;flex-direction:column;-webkit-box-pack:center;-ms-flex-pack:center;justify-content:center;-webkit-box-align:center;-ms-flex-align:center;align-items:center;height:100%;width:100%;background-color:rgba(169,169,169,0.9)}.log{display:-webkit-box;display:-ms-flexbox;display:flex;-webkit-box-orient:vertical;-webkit-box-direction:normal;-ms-flex-direction:column;flex-direction:column;-webkit-box-align:center;-ms-flex-align:center;align-items:center}.log-header{padding-bottom:10px}.log-wrapper{padding:30px 0}.log-text{padding:10px}.log-text-container{width:100%;height:500px;display:-webkit-box;display:-ms-flexbox;display:flex;overflow:scroll;color:limegreen;background-color:black}
```

Comment:
There's a school of thought that says that checking compiled assets into git is something to avoid. In this case, it will mean that you have a conflict in this file any time you change 2 different scss files on 2 different branches, and you won't be able to resolve it in the github "resolve conflicts" ui because it will need recompiling on your machine.

I think it's fine for now, but once you have some build instructions in your README, you should probably delete this file and stiles.min.css.map, and add them to your `.gitignore`.